README.md

```diff
@@ -689,29 +689,32 @@ $ composer require google/cloud-vision
 ```php
 require 'vendor/autoload.php';
 
-use Google\Cloud\Dlp\V2beta1\DlpServiceClient;
-use Google\Cloud\Dlp\V2beta1\ContentItem;
-use Google\Cloud\Dlp\V2beta1\InfoType;
-use Google\Cloud\Dlp\V2beta1\InspectConfig;
+use Google\Cloud\Dlp\V2\DlpServiceClient;
+use Google\Cloud\Dlp\V2\ContentItem;
+use Google\Cloud\Dlp\V2\InfoType;
+use Google\Cloud\Dlp\V2\InspectConfig;
 
 $dlpServiceClient = new DlpServiceClient();
-$name = 'EMAIL_ADDRESS';
-$infoTypesElement = new InfoType();
-$infoTypesElement->setName($name);
-$infoTypes = [$infoTypesElement];
+$infoTypesElement = new InfoType()
+    ->setName('EMAIL_ADDRESS');
 $inspectConfig = new InspectConfig();
-$inspectConfig->setInfoTypes($infoTypes);
-$type = 'text/plain';
-$value = 'My email is example@example.com.';
-$itemsElement = new ContentItem();
-$itemsElement->setType($type);
-$itemsElement->setValue($value);
-$items = [$itemsElement];
+    ->setInfoTypes([$infoTypesElement]);
```

Comment:
`;` above should be removed? And elsewhere?

Reply:
Definitely, thank you. Fixed